playground/config/packages/entry/json.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
It looks like this rule is unused.

Reply:
The test itself was incorrect and has been rewritten.